doc/UsingUT.md

```diff
@@ -0,0 +1,99 @@
+<!--
+GENERATED FILE - DO NOT EDIT
+This file was generated by [MarkdownSnippets](https://github.com/SimonCropp/MarkdownSnippets).
+Source File: /doc/mdsource/UsingDoctest.source.md
+To change this file edit the source file and then execute ./run_markdown_templates.sh.
+-->
+
+<a id="top"></a>
+
+# Using Approval Tests With \[Boost\].UT
+
+
+<!-- toc -->
+## Contents
+
+  * [Getting Started With \[Boost\].UT](#getting-started-with-ut)
+    * [Requirements](#requirements)
+    * [New Project](#new-project)
+<!-- endtoc -->
+
+
+
+## Getting Started With \[Boost\].UT
+
+The [\[Boost\].UT](https://github.com/boost-experimental/ut) test framework works well with Approval Tests.
+
+\[Boost\].UT is a single header/single module, macro-free μ(micro)/Unit Testing Framework that requires C++17 / C++20
+
+### Requirements
+
+Approval Tests for \[Boost\].UT requires that a file called `ut.hpp` is found.
+
+Approval Tests needs \[Boost\].UT version 1.1.4 or above.
+
+### Usage example
+
+Add the following two lines to your source code:
+
+<!-- snippet: ut_main -->
+<a id='snippet-ut_main'/></a>
+```cpp
+// main.cpp:
+#define APPROVALS_UT
+#include "ApprovalTests.hpp"
+```
+<sup>[snippet source](/tests/UT_Tests/main.cpp#L1-L5) / [anchor](#snippet-ut_main)</sup>
+<!-- endsnippet -->
+
+Bellow is an example of a call to an approval test inside a \[Boost\].UT test:
+
+<!-- snippet: ut_main_usage -->
```

Comment:
If you would like to create example code specifically for documentation, feel free to add a new sub-directory to examples/

Reply:
I think the documentation snippets for now will suffice. What I liked about UT is that it's so simple to use (and doesn't have macros). If more questions arise I can make a proper example project then.